### ISSUE TITLE
How to get the label file that you used in HistoSegNet(not only ADP database training)?

### ISSUE THREAD

**Issue by ambarion** [lyndonchan/hsn_v1#5]
Hello, lyndonchan!

Sorry for bothering you again. I've already got the ADP database from ADP website. However, the label in the database is not exactly the same as you used in HistoSegNet.

In hsn paper, you use 'the CNN  pre-trained on predicting the 31 HTTs in the third level of the ADP database'. But in hsn code, the classifier has 51 classes. My understanding is that you use some augmentation method, e.g.Hierarchical Binary Relevance, as you mentioned in ADP paper.

Here is the problem, the ADP repo only provides a training database containing ‘ADP_EncodedLabels_Release1_Flat.csv', no 'ADP_EncodedLabels_Release1.csv'. As the htt class definition in [ADP repo](https://github.com/mahdihosseini/ADP/blob/master/src/htt_def.py), I can't train model in <dataset_type == 'ADP-Release1'>, only can in <dataset_type == 'ADP-Release1-Flat'>. The 'Flat' label file has totally different classes from the dataset you used in HistoSegNet. So even I got the image, I can't reproduce your patch-level classification CNN training.

So I wonder if you or your team could provide the other label file, which can support 51 classes training?

**Comment by lyndonchan** [comment]
@ambarion I had to dig around my files a bit to find it, but I think I found the correct file. Can you send me your email address so I can send it over to you? If it's the correct file you need, we'll probably add the file into the ADP repo release for future users.

**Comment by ambarion** [comment]
@lyndonchan Thank you so much!!! Please feel free to send to ambarion1379@gmail.com. I'll give you feedback in time.
